features/step_definitions/common_steps.rb

```diff
@@ -240,6 +240,17 @@
   end
 end
 
+Then /^I see '(.*)' in the '(.*)' summary table$/ do |content, table_heading|
+  result_table_location = "//*[@class='summary-item-heading'][normalize-space(text())='#{table_heading}']/following-sibling::table[1]"
+  result_table_rows_location = result_table_location + "/tbody/tr[@class='summary-item-row']"
+  result_table_rows = all(:xpath, result_table_rows_location)
+  content_found = false
```

Comment:
Nice! You could do something like the below instead of lines 247-251, but it's up to you!
  
`result_table_rows.any? {|row| row.text.include? content}.should be true`